data/en/rsi_transformation.json

```diff
@@ -0,0 +1,576 @@
+                                    "offset_by": {
+                                        "days": 20
+                                    }
+                                }
+                            }
+                        ]
+                    }]
+                },
+                {
+                    "type": "Question",
+                    "id": "total-turnover-block",
+                    "questions": [{
+                        "guidance": {
+                            "content": [{
+                                    "title": "Include",
+                                    "list": [
+                                        "commission and fees",
+                                        "exports"
+                                    ]
+                                },
+                                {
+                                    "title": "Exclude",
+                                    "list": [
+                                        "VAT",
+                                        "excise duties",
+                                        "grants and subsidies"
+                                    ]
+                                }
+                            ]
+                        },
+                        "answers": [{
+                            "id": "total-turnover-answer",
+                            "label": "Total turnover excluding VAT",
+                            "mandatory": false,
+                            "q_code": "20a",
+                            "type": "Currency",
+                            "currency": "GBP",
+                            "decimal_places": 2,
+                            "default": 0
+                        }],
+                        "id": "total-turnover-question",
+                        "title": "For the period {{ format_conditional_date (answers['period-from'], exercise.start_date)}} to {{ format_conditional_date (answers['period-to'], exercise.end_date)}}, how much was {{respondent.trad_as_or_ru_name}}’s total turnover <em>excluding VAT</em>?",
+                        "type": "General"
+                    }],
+                    "title": "Total turnover excluding VAT"
+                },
+                {
+                    "type": "ConfirmationQuestion",
+                    "title": "Total turnover excluding VAT",
+                    "id": "confirm-zero-turnover-block",
+                    "skip_conditions": [{
+                        "when": [{
+                            "id": "total-turnover-answer",
+                            "condition": "greater than",
+                            "value": 0
+                        }]
+                    }],
+                    "questions": [{
+                        "type": "General",
+                        "answers": [{
+                            "type": "Radio",
+                            "id": "confirm-zero-turnover-answer",
+                            "q_code": "d20a",
```

Comment:
Also guessing slides are out of date with this qcode as its d20 on slides

Reply:
These are dummy codes. Used d20a as it's confirmation of question 20a. Otherwise we would have two questions with same q_code.